### PR TITLE
add support for width: :full and height: :full

### LINKED ISSUE #638: Add support for `width-full`
### Problem

System arguments do not support `width-full`.

### Proposed solution

Map `width: :full` to `width-full`.

### PR DESCRIPTION
**Pull request by joelhawksley**
[Closes #638]

### CODE REVIEW

**Review by manuelpuyol — COMMENTED** [review]
Instead of adding the `:full` key, could we move this to use `Primer::Classify::Utilities`?

**Comment by joelhawksley** [comment]
@manuelpuyol I just tried that and ran into issues with needing `width` and `height` to support falling back to integer values. 

Is there a way to support that in Utilities? If not, I guess I could add a special case to Classify for the integers and let Utilities handle the mapping to CSS.

**Comment by manuelpuyol** [comment]
> I guess I could add a special case to Classify for the integers and let Utilities handle the mapping to CSS.

I think this would be the ideal case 🤔

File: lib/primer/classify/utilities.yml [hunk 1]
```diff
@@ -1269,3 +1269,13 @@
...
+:height:
+  :fit:
+  - height-fit
+  :full:
+  - height-full
+:width:
+  :fit:
+  - width-fit
+  :full:
+  - width-full
```
Comment (manuelpuyol) [inline lib/primer/classify/utilities.yml]: did you add those manually? I think you are supposed to add the keys to https://github.com/primer/view_components/blob/ce6e757ca91729f201faeee641ef7967568ba5fa/lib/tasks/utilities.rake#L11-L21 and run the task :)

**Review by manuelpuyol — APPROVED** [review]

**Pull request merged**